core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -181,7 +184,8 @@ public void hit(Bullet b, float x, float y){
         }
 
         for(int i = 0; i < lightning; i++){
-            Lightning.create(b, lightningColor, lightningDamage < 0 ? damage : lightningDamage, b.x, b.y, Mathf.random(360f), lightningLength + Mathf.random(lightningLengthRand));
+            float litA = b.rotation() + Mathf.range(lightningCone/2) + lightningAngle;
```

Comment:
This should go in the function itself for consistency. No extra variable.

Reply:
Is there a button for accepting suggested requests, or is it committed already?

Reply:
There is no suggested code change or commit. It's a verbal request.

Reply:
Ok